.ci.yaml

```diff
@@ -4033,6 +4044,18 @@ targets:
         ["devicelab", "ios", "mac"]
       task_name: flutter_gallery__transition_perf_e2e_ios
 
+  - name: Mac_build_test flutter_gallery__transition_perf_e2e_ios
+    recipe: devicelab/devicelab_drone_build_test
+    bringup: true # New target: https://github.com/flutter/flutter/pull/111164
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "ios", "mac"]
+      task_name: flutter_gallery__transition_perf_e2e_ios
+      drone_dimensions: >
+        ["device_os=iOS-16","os=Mac-12", "cpu=x86"]
```

Comment:
Why is this necessary if it's `mac_build_test`?
If it is necessary, should they be properties?
```
      os: Mac-12
      cpu: x86
      device_os: iOS-16
```